public/assets/styles/login.scss

```diff
@@ -0,0 +1,255 @@
+%style-button {
+  border: 0;
+  font-size: 0.875em;
+  height: 3.5714em;
+}
+
+%input-style {
+  border: 0;
+  border-bottom: 0.0714em solid $line-color;
+  border-radius: 0;
+}
+
+.login-form{
+  background-color: $color-secondary;
+  border-bottom-left-radius: 0.714em;
+  border-top-left-radius: 0.714em;
+  padding: 5.714em 2.5em;
+
+  h4{
+    color: $color-accent;
+    font-size: 1.5em;
+    font-weight: bold;
+  }
+
+  h5{
+    color: $color-accent;
+    font-size: 1.125em;
+    font-weight: bold;
+    margin-top: 5.625em;
+  }
+}
+
+.signup{
+  background-color: $color-accent;
+  border-bottom-right-radius: 0.625em;
+  border-top-right-radius: 0.625em;
+  color: $white;
+  padding: 5em 2.5em;
+}
+
+.google-btn{
+  @extend %style-button;
+  background-color: $color-google;
+
+  &:hover {
+    background-color: $color-google;
+  }
+}
+
+.facebook-btn{
+  @extend %style-button;
+  background-color: $color-facebook;
+
+  &:hover {
+    background-color: $color-facebook;
+  }
+}
+
+.twitter-btn{
+  @extend %style-button;
+  background-color: $color-twitter;
+
+  &:hover {
+    background-color: $color-twitter;
+  }
+}
+
+
+.email-input {
+  @extend %input-style;
+  background-color: transparent;
+
+  &::placeholder {
+    color: $login-placeholder-color;
+  }
+
+  &:focus {
+    @extend %input-style;
+    background-color: transparent;
+    box-shadow: none;
+  }
+}
+
+.far {
+  opacity: 0.25;
+}
+
+
+.email-addon{
+  @extend %input-style;
+}
+
+.transparent{
+  background-color: transparent;
+}
+
+.login-btn{
+  border-radius: 1.5625em;
+  border-width: 0.0625em;
+  margin-top: 3rem;
+}
+
+.signup-btn{
+  border: 0.125em solid $white;
+  border-radius: 1.5625em;
+  color: $white;
+
+  &:hover {
+    border-color: $white;
+  }
+}
+
+.login-banner{
+  margin-top: 44%;
+}
+
+.welcome-banner{
+  margin-top: 3rem;
+}
+
+.welcome-text {
+  font-size: 2.25em;
+  font-weight: bold;
+  -webkit-text-stroke: 0.03em $white;
+}
+
+h2, h5, h4{
+  font-family: 'Zilla Slab';
+}
+
+p{
+  font-family: 'Open Sans';
+}
+
+.login-wrapper {
+  margin-top: 3rem;
+}
+
+.account-question {
+  font-family: 'Zilla Slab';
+  margin-top: 18%;
+}
+
+.welcome-body {
+  margin-top: 10%;
+}
+
+.social-button {
+  margin-top: 5%;
+}
+
+
+.login-form-section {
+  margin-top: 3rem;
+}
+
+@media only screen and (max-width: 768px) {
+  .account-question {
+    font-size: 90%;
+    margin-top: 80%;
+  }
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
can't all the styles under this media query also fall under the media query on line 157? I believe it can.

Reply:
No, it cant. I want a particular query at that point. That was why I included that query.

Reply:
The difference between the two media queries is 1px. I think this can be done. And whatever problems you may be having with your margins and font-size can be fixed by using em's for your dimensions and not percentages.